src/cargo/ops/cargo_output_metadata.rs

```diff
@@ -109,7 +109,7 @@ where
         .iter()
         .map(|id| Node {
             id,
-            dependencies: resolve.deps(id).collect(),
+            dependencies: resolve.deps(id).map(|p| p.0).collect(),
```

Comment:
Hm, we really need to surface information about renames into the metadata. A backwards compatible way to do that would be to add a new array, `dependency_info`, alongside `dependencies`, with the same length.

Reply:
I'll do that!